net/dnsdist/files/dnsdist.init

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2015 OpenWrt.org
```

Comment:
Source?

Reply:
I've removed this line as I wrote this init script myself.